_articles/manage-repos.md

```diff
@@ -17,10 +19,16 @@ section: pop-ubuntu
 
 ### Accessing Repoman
 
-The tool that Pop!_OS has for adding, removing and editing repositories is called <u>Repoman</u> which is a part of the Pop!_Shop.
+Pop!_OS' tool for adding, removing and editing repositiories is called <u>Repoman</u> which is a part of the Pop!_Shop.
 
 ![Pop!_OS Shop](/images/manage-repos/Pop!_Shop.png)
 
-Once <u>Repoman</u> is opened click on the **Extra Sources** button from this tab there will be options to add repositories with the **Plus** button on the bottom left and edit repositories with the **Edit** button next to the **Plus** button once a repository is selected.
+Once <u>Repoman</u> is opened, click on the **Extra Sources** tab there will be options to add repositories with the **Plus** button (on the bottom left). Select a repository from the list and then click on the **Edit** button (next to the **Plus** button) to edit the selected repository.
 
 ![Repoman window](/images/manage-repos/Repoman.png)
+
+Ubuntu's tool for adding, removing and editing repositories is called <u>Software & Updates</u>.
+
+![Software & Updates](/images/manage-repos/Software&Updates.png)
+
+Once <u>Software & Updates</u> is opened, click on **Other Software** tab there will be options to add repositories with the **Add...** button. Select a repository from the list and then click on the **Edit...** button (next to the **Add...** button) to edit the selected repository.
```

Comment:
Once <u>Software & Updates</u> is opened, click on **Other Software** tab. Under this tab, add repositories with the **Add...** button. Select a repository from the list and then click on the **Edit...** button (next to the **Add...** button) to edit the selected repository.